client/src/screens/Register/Register.jsx

```diff
@@ -1,51 +1,23 @@
 import React, { useState } from 'react';
 
-import {
-  Button,
-  PasswordField,
-  TextField,
-  Paper,
-  Title,
-  Text,
-  Link,
-  Alert,
-  Snackbar,
-  PageContainer,
-} from './Register.styled';
+import { RegisterForm } from '@components';
+import { Alert, Snackbar, PageContainer } from './Register.styled';
 
 export default function Register() {
-  // Will change those states to Formik usage later
-  const [invalidRegister, setInvalidRegister] = useState(false);
-  // If you want to see the Snackbar alert for visual check, change to true.
-  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [userData, setUserData] = useState({});
+  // Just for demonstration, will be used with API request result
+  const goodResponse = true;
 
   return (
     <PageContainer>
-      <Paper variant="elevation" elevation={7}>
-        <Title>Sign Up</Title>
-        <TextField label="Full name" />
-        <TextField label="Phone number" />
-        <TextField label="Username" required />
-        <PasswordField label="Password" />
-        <Text>
-          {'Already have an account ? '}
-          <Link href="http://localhost:3030/login" underline="hover">
-            click here
-          </Link>
-        </Text>
-        <Button label="Register" sx={{ padding: '0.7em' }} fullWidth />
-      </Paper>
+      <RegisterForm onSubmit={(data) => setUserData(data)} />
       <Snackbar
         anchorOrigin={{ vertical: 'bottom', horizontal: 'center' }}
         autoHideDuration={6000}
-        onClose={() => {
-          setIsSubmitted(false);
-          setInvalidRegister(false);
-        }}
-        open={isSubmitted}
+        open={goodResponse}
       >
-        {invalidRegister ? (
-          <Alert severity="error">Invalid registration data</Alert>
+        {goodResponse ? (
```

Comment:
it's look like wrong 
you've chosen goodResponse variable name, while you render here severity error on this case 
please check it again, and give right variable name to this case/alert